README.md

```diff
@@ -1,6 +1,23 @@
 # Knowledge Graph
 
-[GraphML](http://graphml.graphdrawing.org/) is XML based format for describing graphs. This schema is understood by chat-gpt. So if we were to give chat-gpt data input compliant to GraphML, it can answer questions around it. GraphML XML-based syntax supports the entire range of possible graph structure constellations including directed, undirected, mixed graphs, hypergraphs, and application-specific attributes
+## Motivation
+ACM is a complex product that is split into many components.  Many components are optional and need to be installed separately.  While we have a lot of good documentation, there are plenty of areas which we do not document well.
+1. product documentation does not include certain kinds of details because they are out of scope
+1. documentation/training materials by nature are siloed because each team produces there own
+1. so we usually do not have good grasp of flows that span teams
+1. these interconnected flows can be better expressed using Knowledge Graphs (they are simply understood by the human brain because this is usually how we draw it on a white board!)
+
+This will allow:
+1. support team to resolve problems faster without having to call the domain experts
+1. domain experts if called in will have a easier time to recall complex things they have done in the past because Knowledge graphs helps speedy recollection
```

Comment:
```suggestion
1. domain experts if called in will have an easier time to recall complex things they have done in the past because Knowledge graphs help speedy recollection
```

Reply:
done